en/identity-server/next/docs/apis/organization-apis/restapis/org-application-mgt.yaml

```diff
@@ -0,0 +1,2591 @@
+openapi: 3.0.0
+info:
+  description: >
+    This document specifies an **Application Management RESTful API** for **WSO2 Identity Server**.
+  version: "v1"
+  title: WSO2 Identity Server - Application Management Rest API
+  termsOfService: 'http://swagger.io/terms/'
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: architecture@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+security:
+  - OAuth2: []
+  - BasicAuth: []
```

Comment:
basic auth should not be here in org APIs

Reply:
Removed